docs/src/main/paradox/cluster-client.md

```diff
@@ -295,7 +295,7 @@ based API.
 
 ##### Initial contact points
 
-Instead of configured initial contact points the [Pekko gRPC Service Discovery](https://doc.akka.io/docs/akka-grpc/current/client/configuration.html#using-akka-discovery-for-endpoint-discovery) can be used.
+Instead of configured initial contact points the [Pekko gRPC Service Discovery]($pekko.doc.dns$/docs/pekko-grpc/current/client/configuration.html#using-akka-discovery-for-endpoint-discovery) can be used.
 
```

Comment:
pekko-discovery

Reply:
As far as I understand, we need to refer https://github.com/apache/incubator-pekko-grpc/blob/main/docs/src/main/paradox/client/configuration.md#using-akka-discovery-for-endpoint-discovery from this location, this should be access by `pekk-grpc` right? 

or Am I miss understanding something?

Reply:
`#using-akka-discovery-for-endpoint-discovery` should be `#using-pekko-discovery-for-endpoint-discovery`

Reply:
Okay, I'll change it. `#` link will not work until we change in `incubator-pekko-grpc` project